src/encoded/helpers/helper.py

```diff
@@ -103,8 +103,10 @@ def search_result_actions(request, doc_types, es_results, position=None):
 
 class View_Item:
     def __init__(self, request, search_base):
+        # strip out x.limit=<number> and y.limit=<number>; as they are not used
+        search_base_without_limit = re.sub(r'&[x|y]\.limit\=\d*', '', search_base, flags=re.IGNORECASE)
```

Comment:
Do we always want to throw out x limit or just when it's empty?  

Also, we only want this to happen the limit comes after an ampersand?  If it's the first parameters limit will comes after a question mark.

Reply:
Good questions. 

We only want to get rid of x/y limit when they are not needed. So, the variable: "search_base_without_limit" is used in some but not all cases (see lines after 107) 

Yes, x/y limit always come after an ampersand. In the narrow case where this code is used, x/y limit are never the first parameter. The first parameter is typically "type", like "type=Experiment"

Reply:
The ticket is not clear on whether we want to always remove limit or only when it breaks the site (i.e. empty limit).  Please check with Ben and update the JIRA to be more clear.  Do we always remove it in certain circumstances or only when it's empty.  To me the problem is that 'limit=' where limit=1 doesn't.  But that is a product question.

***

I don't see a good reason to always assume limit is after &.  Seems like an unnecessary qualification.

Reply:
per convo, I was misunderstanding the feature.  But we did decided to make is ? or &

Reply:
also seems to complicated to fix, so we're assuming x limit is not first.